File: 0945-minimum-increment-to-make-array-unique/0945-minimum-increment-to-make-array-unique.java
class Solution {
    public int minIncrementForUnique(int[] nums) {
        int ans = 0;
        Arrays.sort(nums);
        int num = nums[0];
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == num) {
                num++;
                continue;
            }
            else if (nums[i] > num) {
                num = nums[i] + 1;
                continue;
            }
            else {
                ans += num - nums[i];
                num += 1;
            }
        }
        return ans;
    }
}

File: 0945-minimum-increment-to-make-array-unique/0945-minimum-increment-to-make-array-unique.py
class Solution:
    def minIncrementForUnique(self, nums: List[int]) -> int:
        ans = 0
        nums = sorted(nums)
        num = nums[0]
        for i in range(0, len(nums)):
            if nums[i] == num:
                num += 1
                continue
            elif nums[i] > num:
                num = nums[i] + 1
                continue
            else:
                ans += num - nums[i]
                num += 1
        return ans